<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationsDashboardControlWidgetClass</class>
 <widget class="QMainWindow" name="OperationsDashboardControlWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OperationsDashboardControlWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TV Screen Controlpanel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,3,3,3,0">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="gB_Status">
        <property name="title">
         <string>Status to be Displayed</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="cB_New">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cB_InProgress">
           <property name="text">
            <string>In Progress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cB_ServiceScheduled">
           <property name="text">
            <string>Sercive scheduled</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cB_WaitingForParts">
           <property name="text">
            <string>Waiting for Parts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cB_WaitingforProductionsResponse">
           <property name="text">
            <string>Waiting for production response</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cB_Resolved">
           <property name="text">
            <string>Resolved</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cB_Closed">
           <property name="text">
            <string>Closed</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Font Size</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Current Font Size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_currentFontSizeData">
               <property name="text">
                <string>8pt</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pb_decrease">
               <property name="text">
                <string>decrease</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_increase">
               <property name="text">
                <string>increase</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Fullscreen display</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Set fullscreen Display</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cB_screens"/>
              </item>
              <item>
               <widget class="QPushButton" name="pB_setfullscreen">
                <property name="text">
                 <string>Set Fullscreen</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Scroll Speed</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="bottomMargin">
            <number>50</number>
           </property>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="dSB_SpeedUpPxPerSec">
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="dSB_SpeedDownPxPerSec">
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Speed Down</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Speed Up</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sB_PauseBottomMs">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sB_PauseTopMs">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Pause Bottom</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Pause Top</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
